bridge/slack/slack.go

```diff
@@ -711,19 +716,20 @@ func (s *Slack) sendPublicMessage(ghost *bridge.UserInfo, msg, channelID string)
 func (s *Slack) handleSlackActionPost(rmsg *slack.MessageEvent) {
 	logger.Debugf("handleSlackActionPost() receiving msg %#v", rmsg)
 
-	hasOurCallbackID := false
-	if len(rmsg.Blocks.BlockSet) == 1 {
-		block, ok := rmsg.Blocks.BlockSet[0].(*slack.SectionBlock)
-		hasOurCallbackID = ok && block.BlockID == "matterircd_"+s.sinfo.User.ID
-	}
+	// Is this our own message
+	if rmsg.User == s.sinfo.User.ID {
+		lastmsg := s.msgLast[rmsg.Channel]
 
-	if rmsg.SubMessage != nil && len(rmsg.SubMessage.Blocks.BlockSet) == 1 {
-		block, ok := rmsg.SubMessage.Blocks.BlockSet[0].(*slack.SectionBlock)
-		hasOurCallbackID = ok && block.BlockID == "matterircd_"+s.sinfo.User.ID
-	}
+		// Slack can be faster in sending new message than replying to POST
+		if lastmsg < rmsg.Timestamp {
+			time.Sleep(100 * time.Millisecond)
```

Comment:
is 100ms enough here? shouldn't this be in a for loop?

Reply:
10ms was not enough, 100ms works for me... Didn't do any more measurements.
for loop (or rather infinite wait) wouldn't work here since it is possible to write messages with web slack and then it is normal to have lastmsg < rmsg.Timestamp.